.github/ISSUE_TEMPLATE/icon_request.yml

```
@@ -0,0 +1,85 @@
+name: Icon request
+description: Request a new icon for Simple Icons
+title: "Request: "
+labels: [new icon]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        We won't add non-brand icons or anything related to illegal services. If in
+        doubt, open an issue and we'll have a look. Before opening a new issue, make
+        sure it isn't covered by an existing issue. Please search for [issues with the
+        `new icon`
+        label](https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22new+icon%22+is%3Aopen)
+        (including closed issues) before you continue. If you find one for the brand
+        you're requesting then leave a comment on it or add a reaction to the original
+        post. Also take a look at our [contributing
+        guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md)
+        for more details.
+
+  - type: input
+    attributes:
+      label: Brand Name
+      placeholder: "Example: Simple Icons"
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Website
+      description: >-
+        For non-web brands you can add a relevant link or simply put "None" if you don't
+        think there's a website.
+      placeholder: "Example: https://simpleicons.org"
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Popularity Metric
+      description: >-
+        Provide either an [Alexa rank](https://www.alexa.com/siteinfo) or the number
+        of GitHub stars. If these are not available or insufficient you can still open
+        the issue. Please provide any information regarding the brand's popularity you
+        think is relevant in the description below.
+      placeholder: "Example: 249,089"
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Official resources for icon and color
+      description: >-
+        Media kits, brand guidelines, SVG files, etc. We strongly encourage you to do
+        some research here as it will help implement your request faster, although you
+        can set this to none if you are unable to find any resources.
+      placeholder: |
+        Example:
+        svg source: https://github.com/simple-icons/simple-icons/blob/develop/icons/simpleicons.svg
+        color: #111111 from website header
```

Minor, but we can try to encourage people to take advantage of GitHub's rendering capabilities of colors (e.g. `#123456`):

```suggestion
        color: `#111111`, from the website header
```

(if this doesn't work with YAML then just ignore this comment :upside_down_face:)

---

Works fine, but the placeholders are not markdown rendered.